src/components/Containers/Accounts/AccountsList/AccountDetails.js

```diff
@@ -54,14 +55,9 @@ const AccountDetails = ({ account, asHeader, actions }) => {
     return formatNumber(diff);
   };
 
-  const renderBadge = () => {
-    let badgeStatus;
-    if (asHeader) return null;
-    if (status === 'Created') return null;
-    if (status === 'Importing') badgeStatus = 'normal';
-    if (status === 'Complete') badgeStatus = 'positive';
-    if (status === 'Failed') badgeStatus = 'caution';
-    return <Badge name={status} type={badgeStatus} />;
+  const getHiddenStatus = () => {
+    if (status === STATUSES.IM || status === STATUSES.RE) return true;
+    return false;
```

Comment:
for what it's worth, this can be a one-liner:
`const getHiddenStatus = () => status === STATUSES.IM || status === STATUSES.RE;`